Sources/Socket.swift

```diff
@@ -1,5 +1,28 @@
+#if os(Linux)
 import Glibc
 
+private let sock_stream = SOCK_STREAM.rawValue
+
+private let system_accept = Glibc.accept
+private let system_bind = Glibc.bind
+private let system_close = Glibc.close
+private let system_listen = Glibc.listen
+private let system_read = Glibc.read
+private let system_send = Glibc.send
+private let system_shutdown = Glibc.shutdown
+#else
+import Darwin.C
+
+private let sock_stream = SOCK_STREAM
+
+private let system_accept = Darwin.accept
+private let system_bind = Darwin.bind
+private let system_close = Darwin.close
+private let system_listen = Darwin.listen
+private let system_read = Darwin.read
+private let system_send = Darwin.send
+private let system_shutdown = Darwin.shutdown
```

Comment:
Perhaps it would be simpler to:

```
typealias System = Darwin
```

Then use System everywhere, What do you think?